django/contrib/postgres/constraints.py

```diff
@@ -0,0 +1,90 @@
+from django.db.models import CheckConstraint, F
+from django.db.models.sql import Query
+
+__all__ = ['ExcludeConstraint']
+
+
+class ExcludeConstraint(CheckConstraint):
+    template = 'CONSTRAINT %(name)s EXCLUDE USING %(index_type)s(%(expressions)s)%(where)s'
+
+    def __init__(self, *, expressions=(), index_type='GIST', operators=(), check=None, name=''):
```

Comment:
From what I can read from PostgreSQL's documentation there's no way to define a check for an exclusion constraint?

https://www.postgresql.org/docs/11/static/sql-createtable.html#SQL-CREATETABLE-EXCLUDE

I'd say we drop this argument completely.